drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1597,6 +1620,12 @@ function DragAndDropBlock(runtime, element, configuration) {
                 } else {
                     revertDrag();
                 }
+                if (configuration.display_zone_borders_dragging) {
+                    if (last_target_zone) {
+                        last_target_zone.removeClass('zone-border');
+                        last_target_zone = null;
+                    }
+                }
```

Comment:
```suggestion
                if (configuration.display_zone_borders_dragging && last_target_zone) {
                        last_target_zone.removeClass('zone-border');
                        last_target_zone = null;
                }
```

Wouldn't this be cleaner?

Reply:
Fixed